core.js

```diff
@@ -1139,6 +1139,19 @@ export class FileTypeParser {
 			}
 		}
 
+		if (
+			this.check([0x57, 0x45, 0x42, 0x56, 0x54, 0x54, 0x00])		// EOF
+			|| this.check([0x57, 0x45, 0x42, 0x56, 0x54, 0x54, 0x0A])	// LF
+			|| this.check([0x57, 0x45, 0x42, 0x56, 0x54, 0x54, 0x0D])	// CR
+			|| this.check([0x57, 0x45, 0x42, 0x56, 0x54, 0x54, 0x09])	// Tab
+			|| this.check([0x57, 0x45, 0x42, 0x56, 0x54, 0x54, 0x20])	// Space
+		) {
+			return {
+				ext: 'vtt',
+				mime: 'text/vtt',
+			};
+		}
```

Comment:
Is this a sensible way of going about it, or should I be testing the common byte sequence first and then the last byte in a nested statement? Also, does it make sense to have separate fixtures for these different characters after the initial `WEBVTT`?

Reply:
That makes sense, thanks! I'll make these changes.

Btw, is there any code formatter set up for this project? I couldn't find anything in the docs or package.json. Fixing the tests is a bit frustrating as it's getting hung up on code formatting.

Reply:
```
npx xo --fix
```